node-util/sbin/oo-init-quota

```diff
@@ -79,11 +79,11 @@ QUOTA_OPTIONS=usrjquota=aquota.user,jqfmt=vfsv0
 function update_fstab() {
     # ORIGIN_MOUNTPOINT=$1
 
-    if ! tr -s ' ' < /etc/fstab | egrep "^[^#].+ $1 ext4 [^ ]+$QUOTA_OPTIONS "; then
```

Comment:
Interesting: `[^ ]+` causes this old regex to fail to match if the quota options are not preceded by other options, so this change avoids adding duplicate `$QUOTA_OPTIONS` in that case.